front-end-dep-tree/EVALUATION.md

```diff
@@ -0,0 +1,19 @@
+## Review Questions
+
+These are the questions that we should ask when reviewing a homework submission.
+
+* Is the app in one of the languages listed in the instructions? If not, did the applicant get special permission to use something else?
+* Does the submittions include instructions for getting the assgignment up and running locally.
+* Does the app meet each of the user story requirements set in the assignment
+* Is the code well organized?
+  * Is it split into multiple methods and views in an appropriate way?
+  * Is there a clear split between the command line program and libraries?
```

Comment:
It's no longer a command line app. A program/library split probably isn't that relevant anymore either

Reply:
Fixed